packages/react-core/src/components/Page/Page.tsx

```diff
@@ -107,7 +108,9 @@ export class Page extends React.Component<PageProps, PageState> {
   componentDidMount() {
     const { isManagedSidebar, onPageResize } = this.props;
     if (isManagedSidebar || onPageResize) {
-      window.addEventListener('resize', debounce(this.handleResize, 250));
+      window.addEventListener('resize', this.handleResize);
+      document.addEventListener('mousedown', this.handleClickMobile);
```

Comment:
these could be attached to the ref, something like
```suggestion
      this.outerRef && this.outerRef.current && this.outerRef.current.addEventListener('mousedown', this.handleClickMobile);
```

Reply:
Sure, although we lose this feature for clicking below pages that don't take up `100vh`.